### DISCUSSION TITLE
Point task to a specifc worker or cluster

### DISCUSSION THREAD

**Discussion by stiliajohny** [concourse/concourse#6364] [Help & Support] [answered]
hi all, 

I am currently have the following infra 

**Pod :** k8s cluster
**Stage :** k8s cluster
**Dev :** k8s cluster

Each environment belongs to a different AWS account and all k8s clusters can see each other from a network perspective.

My end goal is to run a pipeline that will effectively run one job on dev cluster then I can trigger(or autorun) next job to the stage cluster, then run the next job on the prod cluster. 

Many thanks in advance,

**Comment by taylorsilva** [comment, marked as answer]
Sorry for the late reply! Tags are what you want: https://concourse-ci.org/jobs.html#schema.step.tags

**Reply by stiliajohny** [reply]
Legendary... 
I will have a look.
Thanks a lot.
